ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -150,6 +150,9 @@ keys:
             keys:
               d_path:
                 type: bool
+          redistribute_internal:
```

Comment:
I would expect the same keys to be needed under the VRFs and the VRF ipv4/ipv6 address-families.

Reply:
Fixed.